Rubberduck.Core/UI/UnitTesting/TestExplorerViewModel.cs

```diff
@@ -132,6 +139,43 @@ public TestExplorerGrouping TestGrouping
             }
         }
 
+        //public System.Collections.ObjectModel.ObservableCollection<System.Windows.Media.Imaging.BitmapImage> OutcomeFilters { get; }
+        public System.Collections.ObjectModel.ObservableCollection<object> OutcomeFilters { get; }
```

Comment:
Why the full qualification?

Reply:
Was developing without R#. Too much time passed and hadn't remembered 😞